ModSupport/ModCalls.cs

```diff
@@ -2507,7 +2506,7 @@ bool castID(object o, out int id)
                 case "CanFirePointBlank":
                 case "CanFirePointBlankShots":
                     if (args.Length < 2)
-                        return new ArgumentNullException("ERROR: Must specify an Item object (or int index of an Item in the Main.item array).");;
+                        return new ArgumentNullException("ERROR: Must specify an Item object (or int index of an Item in the Main.item array)."); ;
```

Comment:
Your auto formatter doesn't catch double semicolons? Interesting

Reply:
It would be a linter's job to flag, a formatter wouldn't necessarily care. Empty semicolons are useful for debugging and stuff.